mt_dnn/inference.py

```diff
@@ -0,0 +1,20 @@
+from data_utils.metrics import calc_metrics
+
+def eval_model(model, data, metric_meta, use_cuda=True, with_label=True, label_mapper=None):
+    data.reset()
+    if use_cuda:
+        model.cuda()
+    predictions = []
+    golds = []
+    scores = []
+    ids = []
+    metrics = {}
+    for batch_meta, batch_data in data:
+        score, pred, gold = model.predict(batch_meta, batch_data)
+        predictions.extend(pred)
+        golds.extend(gold)
+        scores.extend(score)
+        ids.extend(batch_meta['uids'])
+    if with_label:
+        metrics = calc_metrics(metric_meta, golds, predictions, scores, label_mapper)
```

Comment:
A little cconfused , does labels in `golds` already been converted to ids in prepro_std? why do we need label_mapper here?

Reply:
NER is a token level classification task. Since BERT uses sub-word tokens, it says that each word may be split to several tokens. During eval, we need to restore it to word... Thus, we need to ignore many sub-word tokens...